src/Microsoft.AspNet.Html.Abstractions/HtmlContentBuilder.cs

```diff
@@ -1,62 +1,97 @@
-// Copyright (c) .NET Foundation. All rights reserved.
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.IO;
 using System.Text.Encodings.Web;
-using Microsoft.AspNet.Html.Abstractions;
 
-namespace Microsoft.Extensions.Internal
+namespace Microsoft.AspNet.Html.Abstractions
 {
     /// <summary>
-    /// Enumerable object collection which knows how to write itself.
+    /// An <see cref="IHtmlContentBuilder"/> implementation using an in memory list.
     /// </summary>
-    [DebuggerDisplay("{DebuggerToString()}")]
-    internal class BufferedHtmlContent : IHtmlContentBuilder
+    public class HtmlContentBuilder : IHtmlContentBuilder
```

Comment:
This is mostly copy-paste. Also some general improvements here because this is a public API now.